README.md

```diff
@@ -1,6 +1,6 @@
 # Portfolio
 
-This is a desktop version of my portfolio.It is done in the template design I have chosen from figma.
+This is a desktop version of my portfolio.It is done in the template design I have chosen from figma.You can see the online version of my portfolio at https://yoz21.github.io/Portfolio/.
```

Comment:
- [ ] Great job deploying your page💯, I'd suggest you make the link easily accessible by adding it as a link👌.
You can try adding it like 
```
## Live Deploy

[Deploy Link](https://yoz21.github.io/Portfolio/)
```

Reply:
thanks